ament_cmake_test/ament_cmake_test/__init__.py

```diff
@@ -72,6 +72,11 @@ def main(argv=sys.argv[1:]):
         action='store_true',
         default=False,
         help='Skip the test')
+    parser.add_argument(
+        '--skip-return-code',
+        type=int,
```

Comment:
Maybe we should explicitly check for the valid range `[0, 255]`.

Reply:
It doesn't appear that CMake checks this. I'd suppose I can add a check and fail the test if an invalid value is specified.

Reply:
153b2d9